### ISSUE TITLE
Problem with Infantry

### ISSUE THREAD

**Issue by marvinblue** [Thalassicus/cep-bnw#254]
Hi there. I've found a problem with the mods that causes a crash. Once I upgrade most of my riflemen to infantry and go to move one of them, the game crashes back to Windows. The units don't have any artwork with them, they're just blank with the icon above them. My machine has an AMD A8-5500 APU with 4 Gb RAM and an AMD Radeon HD 7560D GPU with 2 Gb RAM. I also have all of the most current cep mods upto 14.1.

**Comment by marvinblue** [comment]
I enabled logging, but I can't attach my LUA log?

**Comment by GrantSP** [comment]
Rename it to a **JPG, PNG or GIF**.
GitHub allows only those type as attachments. Don't worry about it not actually being of that type.

**Comment by marvinblue** [comment]
Done! Thanks for that!
![lua](https://cloud.githubusercontent.com/assets/7192736/2624110/9fde993e-bd20-11e3-8545-723f013c3d35.jpg)

**Comment by GrantSP** [comment]
This doesn't sound like a problem in the mod, more like an installation problem.

There haven't been any issues like you describe show up before this current version of 3.15.1
The graphics problems may indicate deeper problems with your setup. Vanilla games have no issues with graphics?
Try uninstalling the mods, clearing the cache and **ModUserData** folder and then re-installing.

The length of your log seems to indicate you perhaps started a savegame and then found this problem, did you change your setup inbetween? Perhaps updated a mod or added something else?

**Comment by marvinblue** [comment]
No, this was a fresh game. Incidentally, it also happened with another game that I started from scratch. I was also playing as Arabia and the same thing happened when I upgraded to infantry. I'll try the above suggestions and see what happens.

**Comment by marvinblue** [comment]
I've just done as requested above. Unfortunately, the game still crashed.

**Comment by GrantSP** [comment]
This doesn't sound like a mod problem.
Try verifying your game installation via the Steam client. Also check you have the latest updated version of the game, it should be version 1.0.3.144.

**Comment by marvinblue** [comment]
I usually launch the game from Steam, so the updates are current, I'm running version 1.0.3.144. I've attached my log and some screenshots as well. Should I either roll-back the mods to 3.13 or upgrade to 3.15? Also, the unit is rifleman, not infantry. I apologise for that.
![lua](https://cloud.githubusercontent.com/assets/7192736/2626985/30300f84-bdf5-11e3-89d6-90a60596072d.jpg)
![2014-04-07_00001](https://cloud.githubusercontent.com/assets/7192736/2626986/3a29b774-bdf5-11e3-9965-a4f7230f0cf3.jpg)
![2014-04-07_00002](https://cloud.githubusercontent.com/assets/7192736/2626987/44f6e49c-bdf5-11e3-95f9-d2fc357e8fc4.jpg)

**Comment by GrantSP** [comment]
Most definitely get the latest version from [here](https://github.com/Thalassicus/cep-bnw/releases/latest), version 3.15.1.
Or you could wait a day or two, there is possibly an imminent release including **ALL** the bugfixes over the last couple of months. :smiley: 

Before you clear your cache, moduserdata, etc and install this later version, could you zip the contents of the **Logs** folder and attach it to a post here?
The _lua.log_ shows no problem with the mod but as the screenshots show, something is not right.

**Comment by marvinblue** [comment]
I might as well wait for your big update. Here's the zip folder. Thanks for all of your help so far. That's everything from the Log folder. I couldn't attach a *.zip, so I converted all of the files then attached them individually.
![achievements_debug](https://cloud.githubusercontent.com/assets/7192736/2627119/eac1598e-bdfa-11e3-842d-e6ecd1902381.jpg)
![app](https://cloud.githubusercontent.com/assets/7192736/2627120/eadf1abe-bdfa-11e3-8295-80984ffc55be.jpg)
![database](https://cloud.githubusercontent.com/assets/7192736/2627121/eb056c3c-bdfa-11e3-808f-c8cf2b352d27.jpg)
![espionagelog](https://cloud.githubusercontent.com/assets/7192736/2627122/eb319f78-bdfa-11e3-93d0-99a40aaffcd9.jpg)
![localization](https://cloud.githubusercontent.com/assets/7192736/2627123/eb39a380-bdfa-11e3-85f3-ec871ca258ac.jpg)
![init_host](https://cloud.githubusercontent.com/assets/7192736/2627124/eb3e7856-bdfa-11e3-9e99-e33d9d2d5f61.jpg)
![modding](https://cloud.githubusercontent.com/assets/7192736/2627125/eb43d80a-bdfa-11e3-8a03-d3165a2d9832.jpg)
![lua](https://cloud.githubusercontent.com/assets/7192736/2627126/eb470dae-bdfa-11e3-8925-792e53e1f969.jpg)
![net_message_debug](https://cloud.githubusercontent.com/assets/7192736/2627127/eb5495d2-bdfa-11e3-9fe8-107e48f80e3b.jpg)
![steamcloud](https://cloud.githubusercontent.com/assets/7192736/2627128/eb5f97a2-bdfa-11e3-8a54-cfec7547a194.jpg)
![stopwatch](https://cloud.githubusercontent.com/assets/7192736/2627129/eb699f5e-bdfa-11e3-86da-2b99217855e8.jpg)
![system_init](https://cloud.githubusercontent.com/assets/7192736/2627130/eb70e214-bdfa-11e3-8536-93689122f34e.jpg)
![traderoutelog](https://cloud.githubusercontent.com/assets/7192736/2627131/eb74b5f6-bdfa-11e3-912d-82a9fac467b1.jpg)
![xml](https://cloud.githubusercontent.com/assets/7192736/2627132/eb79458a-bdfa-11e3-829f-d51b1cc4741f.jpg)
![xml-perf](https://cloud.githubusercontent.com/assets/7192736/2627133/eb7ddf8c-bdfa-11e3-9b64-449d7f7d2254.jpg)

**Comment by GrantSP** [comment]
Ahh.. thanks!?
You do know you could have zipped them and renamed the zip?
Not to worry.
I haven't seen your bug in the postings before so perhaps there is something else happening, at any rate the next release is, if I may say so, a _good_ one.

**Comment by marvinblue** [comment]
Tried renaming, wouldn't work. Possibly my passion fingers..... Sounds like the next update will probably fix it.
